src/arrayMethodSort.js

```diff
@@ -4,8 +4,41 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  // write code here
+  const compareAsStrings = function(a, b) {
+    const aString = String(a);
+    const bString = String(b);
+
```

Comment:
bad variable names
My suggestion:
```suggestion
  const compareAsStrings = function(current, next) {
    const currentString = String(current);
    const nextString = String(next);

```

Reply:
Погоджуюся з коментарем Ані